### DISCUSSION TITLE
Suggestion: Add a page in documentation website to explain the behavior of each command

### DISCUSSION THREAD

**Discussion by andrey-jef** [ObsidianPublisher/obsidian-github-publisher#75] [General]
Hi,

Since I have some ambiguity when experimenting the plugin's command, I would like to suggest a dedicated page in [documentation website](https://obsidian-publisher.netlify.app/) to explain the behavior of each command in the plugin [Github Publisher](https://github.com/ObsidianPublisher/obsidian-github-publisher).

Here is my attempt to clarify the behavior of each command, for your consideration.

`Github Publisher: Share active file` -> should change to `Github Publisher: Upload single current active note`
- Behavior: publish only the single current active note, this command also add the key `share: true` into the frontmatter of the current active note

`Github Publisher: Upload all shared notes` -> change to `Github Publisher: Upload multiple notes`
- Behavior: publish all notes which have the key `share: true` in their frontmatter

`Github Publisher: Upload new shared notes` -> change to `Github Publisher: Upload unpublished notes`
- Behavior: publish only the notes satisfying these two conditions: have the key `share: true` in their frontmatter AND were not published since last upload

`Github Publisher: Check validity of this repository`
- Behavior: check and display the result whether there is a valid connection between the plugin and its attached repository

`Github Publisher: Upload all edited notes since last upload` -> change to `Github Publisher: Refresh content of all published notes`
- Behavior: publish only the notes satisfying these two conditions: have the key `share: true` in their frontmatter AND were recently modified since last upload

`Github Publisher: Upload all new and edited notes since last upload`
- This command is the combination of 2 command `Github Publisher: Upload new shared notes`  and `Github Publisher: Upload all edited notes since last upload`, isn't it? In this case, is it better to remove this command in cause of its ambiguity?
- Behavior: publish only the notes satisfying these two conditions: have the key `share: true` in their frontmatter AND (were newly created OR recently modified since last upload)

**Comment by Mara-Li** [comment]
Please, discuss this in #77
